docs/10-action-optimize-media.md

```diff
@@ -0,0 +1,52 @@
+# Add an action to optimize media
+
+To add a new action in your sonata admin.
+
+![optimize-action.png](img/optimize-action.png)
+
+## Implementation
+
+You have to extend `Sonata\MediaBundle\Controller\MediaAdminController`
```

Comment:
Why don't do it in this bundle? It would be more simple for the end user

Reply:
ok so let's add a Ekino\TinyPngSonataMediaBundle\MediaAdminController using the trait and extending CRUDController. Then you have to explain in the doc that you can use this controller or only the trait (as it is currently explain in the doc)

Reply:
By adding the extension it allows to add or hidden the action button, @bmaziere advised to me to add a Trait & AdminExtension

Reply:
We could enable/disable the action button through the config?

Reply:
We could
<img width="170" alt="Capture d’écran 2019-08-22 à 14 56 19" src="https://user-images.githubusercontent.com/5670968/63516290-06aff980-c4ed-11e9-8503-d479e9591e21.png">

by removing the config below in `sonta_admin.yaml` :
```
    extensions:
        ekino.tiny_png_sonata_media.extension.optimize:
            admins:
                - sonata.media.admin.media
```

Reply:
Yep but in the final project, it would be more friendly if the bundle embeds it and removes it regarding the config

Reply:
@bmaziere extend the `CRUDController` instead of the `Sonata\MediaBundle\Controller\MediaAdminController` ?

Reply:
of course extends Sonata\MediaBundle\Controller\MediaAdminController ;)